pom.xml

```diff
@@ -71,16 +71,16 @@
     <tag>HEAD</tag>
   </scm>
 
-  <distributionManagement>
+  <!--distributionManagement>
```

Comment:
I'm not sure if commenting this out was only for earlier exploratory purposes

Reply:
It was. Fixed by commit 4a9b6234